frontend/src/hooks/queryBuilder/useTag.ts

```diff
@@ -47,9 +49,10 @@ export const useTag = (
 			) {
```

Comment:
Just above this line, there is `isExistsNotExistsOperator(value)`. I am constantly surprised by this kind of logic. If anyone thinks about a moment, it's easy to think where it breaks. This logic allows something like this. No operator no value.
<img width="202" alt="Screenshot 2023-05-06 at 4 23 50 PM" src="https://user-images.githubusercontent.com/22846633/236619986-1fcc0b14-3035-4ff3-b020-ed1d4a0eb6d7.png">

Reply:
@srikanthccv 
https://github.com/SigNoz/signoz/pull/2629/commits/172bc0fb2892e1e3e82db79de0ddba85fc9f2d93

Reply:
That's still not a correct way to fix it. `isExistsNotExistsOperator` should only return true when the operator is `EXISTS` or `NOT_EXISTS` not the simple text search.